### PR TITLE
Improvements to current website

### PR DESCRIPTION
**Pull request by rikaznismi**
## Current changes:
- Fix the inconsistent spacing on homepage, at representative text(compare with current website)(figma)
- Fix border thickness to be more even on faq, before it was very thin when expanding a question(figma)
- Change header to one line, to match figma(figma)
- Change header size on company page, and faq, to be the same(figma)
- Add product catalog button in company page(figma)
- Change buttons in faq to have zoom in when hovered, and brighten up
- Add transition to header, and statistic part on homepage
- Add zoom in for signup button when hovering
- Make carousel smaller in faq 
- Make the first letter in the first paragraph of the homepage to be larger(figma)
- Make the padding at top on headers for every page the same, (200px)
## Font and size were changed to :
- Header: text-5xl, font-medium
- Subheader: text-4xl font-normal

### Feel free to remove or edit any change, the transition in homepage does perhaps not look better.

### CODE REVIEW

**Comment by rikaznismi** [comment]
I will create PR for prod, after it has been accepted

File: src/pages/index.tsx [hunk 1]
```diff
@@ -10,13 +11,21 @@ export default function Home() {
...
   return (
     <>
       <div className="w-full h-full">
         <div className="flex flex-col mx-auto max-w-[75%]">
           <div className="relative lg:py-[150px] py-[75px]">
             <img
-              className="absolute w-full h-[290px]"
+              className={`absolute w-full h-[290px] transition-all duration-[2000ms] ease-in-out ${
+                showLogo ? "opacity-100" : "opacity-0"
+              }`}
```
Comment (ernum) [inline src/pages/index.tsx:28]: I think the transition should be fast. Maybe decrease the duration to 125ms?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -26,16 +35,18 @@ export default function Home() {
             />
 
             {/* <h1 className="absolute text-white drop-shadow-[0_0_10px_rgba(0,0,0,0.25)] text-5xl">
-              11 OKTOBER 2023
-            </h1> */}
+            11 OKTOBER 2023
+          </h1> */}
```
Comment (ernum) [inline src/pages/index.tsx:39]: We can remove this now

**Review by ernum — APPROVED** [review]
LGTM! 🎉

**Pull request merged**